src/client.js

```diff
@@ -124,6 +128,20 @@ let pushClient = {
 
     if (subscription) {
       await subscription.unsubscribe();
+
+      if (endpoint) {
+        // POST subscription details
+        await fetch(endpoint, {
+          method: 'post',
+          body: JSON.stringify({
+            action: 'unsubscribe',
+            subscription: subscription
+          }),
+          headers: {
+            'Content-Type': 'application/json'
+          }
+        });
+      }
     }
 
     await registration.unregister();
```

Comment:
Do we want to unregister the sw?

If the dev uses their own SW, they may want to unsubscribe but keep the sw to be used for fetch events.